src/xrCore/_math.cpp

```diff
@@ -287,6 +289,11 @@ u32 cpufreq()
     sysctlbyname("dev.cpu.0.freq", &cpuFreq, &cpuFreqSz, nullptr, 0);
     return cpuFreq;
 }
+#else
+u32 cpufreq()
+{
+    return 2000;
+}
```

Comment:
Can be done with a similar sysctl on macOS, although it needs a uint64_t instead & gives frequency in Hz.

```suggestion
u32 cpufreq()
{
    u64 cpuFreq = 0;
    size_t cpuFreqSz = sizeof(cpuFreq);
    sysctlbyname("hw.cpufrequency", &cpuFreq, &cpuFreqSz, nullptr, 0);
    return static_cast<u32>(cpuFreq / 1000000);
}
```
Could be combined with the FreeBSD code above it for less redundancy.